docs/configuration/model/ModelConfiguration.md

```diff
@@ -179,3 +179,9 @@ scenarioPropertiesConfig {
 }
 ```
 Configuration is similar to [component configuration](#configuration-of-ui-attributes-of-components)
+
+The docs button which is displayed in the modal window of a component is by default disabled for properties. One can enable it by adding
```

Comment:
"scenario properties modal window" or "modal opened after clicking properties button" - this is not a "modal window of a COMPONENT"

Reply:
I have changed the wording